pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -277,11 +277,22 @@ type CertificateStatus struct {
 	// +optional
 	LastFailureTime *metav1.Time `json:"lastFailureTime,omitempty"`
 
+	// The time after which the certificate stored in the secret named
+	// by this resource in spec.secretName is valid.
+	// +optional
+	NotBefore *metav1.Time `json:"notBefore,omitempty"`
```

Comment:
Because we need to round-trip between all API versions. It's not possible to add a field to a newer api version without adding some way to represent them in the older. New versions are created when you make breaking changes to the _wire format_ (i.e. renaming or moving a field). Not when adding new fields 😄